src/Database/Traits/Purgeable.php

```diff
@@ -54,26 +54,24 @@ public function addPurgeable($attributes = null)
 
     /**
      * Removes purged attributes from the dataset, used before saving.
-     * @param $attributes mixed Attribute(s) to purge, if unspecified, $purgable property is used
+     * @param array|string $attributesToPurge Attribute(s) to purge, if unspecified, $purgable property is used
      * @return array Current attribute set
      */
     public function purgeAttributes($attributesToPurge = null)
     {
         if ($attributesToPurge !== null) {
             $purgeable = is_array($attributesToPurge) ? $attributesToPurge : [$attributesToPurge];
-        }
-        else {
+        } else {
             $purgeable = $this->getPurgeableAttributes();
         }
 
         $attributes = $this->getAttributes();
         $cleanAttributes = array_diff_key($attributes, array_flip($purgeable));
         $originalAttributes = array_diff_key($attributes, $cleanAttributes);
 
-        if (is_array($this->originalPurgeableValues)) {
+        if (count($this->originalPurgeableValues)) {
```

Comment:
Can this property ever be anything that isn't countable()? Why was this change needed?

Reply:
No, it'll always be an array.